apps/core/src/utils/transaction/getBalanceChangeSummary.ts

```diff
@@ -51,9 +58,23 @@ export const getBalanceChangeSummary = (
 			amount: amount.toString(),
 			recipient: recipientAddress,
 			owner,
+			unRecognizedToken: !normalizedRecognizedPackages.includes(packageId),
 		};
 
 		acc[owner] = (acc[owner] ?? []).concat(summary);
 		return acc;
 	}, balanceChangeByOwner as BalanceChangeByOwner);
 };
+
+export const getRecognizedUnRecognizedTokenChanges = (changes: BalanceChange[]) => {
+	const recognizedTokenChanges = [];
```

Comment:
Although we can pass recognizedPackagesList as a prop here and run the filter here, I think it's better to include the flag `unRecognizedToken` in `getBalanceChangeSummary`  response.